chrome/browser/net/chrome_network_delegate.cc

```diff
@@ -327,6 +330,95 @@ int ChromeNetworkDelegate::OnBeforeURLRequest(
   return rv;
 }
 
+std::string generateBlockedElemhideJavaScript(std::string url) {
```

Comment:
Is there any way we can store JS like this in a resource instead? We could then potentially sync it from adblockpluschrome directly when building, but also it would be just easier to maintain.

Reply:
agree. I am also fine if we land it with the creation of an issue to use it from a resource or even to generate a sequence of bytes as in libadblockplus.